Fix github action 🚀

### Description

build step failed

```
Error: You must specify a session secret to use sessions
    at statelessSessions (/home/runner/work/resume-backend-keystone/resume-backend-keystone/node_modules/@keystone-6/core/session/dist/keystone-6-core-session.cjs.prod.js:50:[11](https://github.com/bbonkr/resume-backend-keystone/actions/runs/4837251145/jobs/8620934312#step:5:12))
    at Object.<anonymous> (/home/runner/work/resume-backend-keystone/resume-backend-keystone/.keystone/config.js:811:52)
    at Module._compile (node:internal/modules/cjs/loader:[12](https://github.com/bbonkr/resume-backend-keystone/actions/runs/4837251145/jobs/8620934312#step:5:13)54:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:[13](https://github.com/bbonkr/resume-backend-keystone/actions/runs/4837251145/jobs/8620934312#step:5:14)08:10)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1[14](https://github.com/bbonkr/resume-backend-keystone/actions/runs/4837251145/jobs/8620934312#step:5:15)1:19)
    at require (node:internal/modules/cjs/helpers:110:[18](https://github.com/bbonkr/resume-backend-keystone/actions/runs/4837251145/jobs/8620934312#step:5:19))
    at loadBuiltConfig (/home/runner/work/resume-backend-keystone/resume-backend-keystone/node_modules/@keystone-6/core/dist/loadConfig-c9279d39.cjs.prod.js:42:32)
    at Object.loadConfigOnce (/home/runner/work/resume-backend-keystone/resume-backend-keystone/node_modules/@keystone-6/core/dist/loadConfig-c9279d39.cjs.prod.js:48:10)
Error: Process completed with exit code 1.
```

--- a/.github/workflows/build-and-tag.yml
+++ b/.github/workflows/build-and-tag.yml
@@ -27,8 +27,6 @@ jobs:
       - name: Build
         run: |
           npm run build
-        env:
-          NODE_ENV: production
 
       - name: get-version
         id: get_version

--- a/.github/workflows/create-release.yml
+++ b/.github/workflows/create-release.yml
@@ -28,8 +28,6 @@ jobs:
       - name: Build
         run: |
           npm run build
-        env:
-          NODE_ENV: production
 
       - name: Release
         id: release_drafter

--- a/.github/workflows/verify-pr.yml
+++ b/.github/workflows/verify-pr.yml
@@ -23,5 +23,3 @@ jobs:
       - name: Build
         run: |
           npm run build
-        env:
-          NODE_ENV: production

